src/main/scala/uniq/networking/mqtt/base/model/MQTTACLRuleName.scala

```diff
@@ -0,0 +1,16 @@
+
+// copyright
+
+package uniq.networking.mqtt.base.model
+
+import io.circe.Encoder
+import io.circe.generic.semiauto.deriveEncoder
+
+final case class MQTTACLRuleName(
+  username: String,
+  topic: String
+)
+
+object MQTTACLRuleName {
+  implicit val encoder: Encoder[MQTTACLRuleName] = deriveEncoder[MQTTACLRuleName]
```

Comment:
`implicit val encoder: Encoder[MQTTACLRuleName] = deriveEncoder` drop `[MQTTACLRuleName]` just `deriveEncoder` is enough.